pkg/cmd/ssh-key/list/list.go

```diff
@@ -66,6 +67,14 @@ func listRun(opts *ListOptions) error {
 	cs := opts.IO.ColorScheme()
 	now := time.Now()
 
+	if t.IsTTY() {
+		t.AddField("TITLE", nil, nil)
+		t.AddField("KEY", nil, nil)
+		t.AddField("AGE", nil, nil)
```

Comment:
What do you think about calling this column "Added" or "Uploaded"? I think that better reflects what is actually being displayed.